app/components/projects/gallery/ProjectsGallery.js

```diff
@@ -1,80 +1,92 @@
 import React from "react";
-// import COMPLEMENTARY from "../../services/galleries/complementary";
-// import ONLINE from "../../services/galleries/online";
-// import DESIGN from "../../services/galleries/design";
-// import HOME_STANGING from "../../services/galleries/homeStaging";
-// import DECOR from "../../services/galleries/decor";
-// import Masonry, { ResponsiveMasonry } from "react-responsive-masonry";
+import Masonry, { ResponsiveMasonry } from "react-responsive-masonry";
+import COMPLEMENTARY from "../../services/galleries/complementary";
+import ONLINE from "../../services/galleries/online";
+import DESIGN from "../../services/galleries/design";
+import HOME_STANGING from "../../services/galleries/homeStaging";
+import DECOR from "../../services/galleries/decor";
 import projectStyles from "../projects.module.css";
-import createCardProject from "../project_menu/createCardProject";
-import SERVICES from "../../services/services";
-import { useState } from "react";
-import ALL from "../../services/galleries/all";
-//https://www.altogic.com/blog/passing-data-from-parent-to-child-in-react
-export default function ProjectsGallery() {
-  const [category1, setCategory1] = useState("");
-  const [category2, setCategory2] = useState("");
-
-  function filterImages() {
-    const images = ALL.filter((image) => {
-      if (image.category1 === category1 && image.category2 === category2) {
-        return image;
-      }
-    });
-
-    console.log(images);
-  }
 
-  function handleCategory1(event) {
-    setCategory1(event.target.value);
-
-    filterImages();
-  }
+export default function ProjectsGallery() {
+  return (
+    <div>
+      <div id={"complementar"}>
```

Comment:
it seems all of these divs share the exact same structure

refactor them into a component please

Reply:
done!